lib/go/README.md

```diff
@@ -132,3 +132,28 @@ if this interval is set to a value too low (for example, 1ms), it might cause
 excessive cpu overhead.
 
 This feature is also only enabled on non-oneway endpoints.
+
+A note about server stop implementations
+========================================
+
+[TSimpleServer.Stop](https://pkg.go.dev/github.com/apache/thrift/lib/go/thrift#TSimpleServer.Stop) will wait for all client connections to be closed after 
+the last received or to be received request to be handled, so the time spent by stop
+ may be too long in some situation:
+* when socket timeout is not set,server will be hanged before all active
+  clients to finish handling the last received or to be received request.
+* when the socket timeout is too long saying one hour, server will be 
```

Comment:
1. `s/saying/say/`
2. `s/will/might/`, also it should be "be hanging" instead of "be hanged", I think?